mlir/lib/Dialect/SCF/Transforms/TileUsingInterface.cpp

```diff
@@ -1993,7 +1936,7 @@ getUntiledConsumerFromSlice(RewriterBase &rewriter,
   }
 
   // 2. Check that the loop is perfectly nested.
-  if (!isPerfectlyNestedForLoops(loops)) {
+  if (!mlir::isPerfectlyNestedForLoops(loops)) {
```

Comment:
```suggestion
  if (!isPerfectlyNestedForLoops(loops)) {
```

Shouldn't be necessary here?